Tasks/PackerBuildV0/src/taskParameters.ts

```diff
@@ -116,13 +116,19 @@ export default class TaskParameters {
 
     private _getAzureADGraphCredentials(connectedService: string): msRestAzure.ApplicationTokenCredentials {
```

Comment:
is this functions still needed? Why not change it to `getGraphCredentials` as done for PackerV1 task?

Reply:
Done